src/content/docs/dropins/user-auth/auth-functions.mdx

```diff
@@ -0,0 +1,301 @@
+---
+title: User Auth Functions
+description: Learn about the API functions provided by the Product Details dropin.
+---
+
+
+## confirmEmail
+
+The `confirmEmail` function uses the supplied `customerEmail` and `customerConfirmationKey` parameters to complete the customer activation process. Adobe Commerce sends the confirmation key to the customer when they request to create an account. The function calls the [`confirmEmail` mutation](https://developer.adobe.com/commerce/webapi/graphql/schema/customer/mutations/confirm-email/).
```

Comment:
I've only been linking the name of the query, mutation, or type. It's a trivial concern, but let me know what you prefer so that we're consistent across dropins.

For example:

```
[`confirmEmail](https://developer.adobe.com/commerce/webapi/graphql/schema/customer/mutations/confirm-email/) mutation
```